testing/test_algorithm/test_tokenize.py

```diff
@@ -43,14 +44,14 @@ def test_tokenize():
     text = 'Иван,родил/девченку-веле¶л 1тащить2пеленку3'
     result = {'пелёнка', '3', 'девчёнка', '2', '1', 'иван'}
     # without dictionary
-    assert tokenize(text, STOP_WORDS) == result
+    assert tokenize(text, STOP_WORDS_NO_LANGUAGE) == result
```

Comment:
It will be good to add second test on another language, English for example

Reply:
Committed update